lib/loaders/babel.js

```diff
@@ -70,7 +70,10 @@ module.exports = {
             if (webpackConfig.useReact) {
                 loaderFeatures.ensurePackagesExistAndAreCorrectVersion('react');
 
-                babelConfig.presets.push(require.resolve('@babel/preset-react'));
+                babelConfig.presets.push([require.resolve('@babel/preset-react'), {
+                    // TODO: To remove when Babel 8, "automatic" will become the default value
+                    runtime: 'automatic',
+                }]);
```

Comment:
Quick-win, let's bump `@babel/preset-react` to ^7.9.0 to use [`runtime: 'automatic'`](https://babeljs.io/docs/babel-preset-react#runtime)